cli/tests/testdata/resolve_dns.ts.out

```diff
@@ -23,3 +23,4 @@ SRV
 TXT
 [["I","am","a","txt","record"],["I","am","another","txt","record"],["I am a different","txt record"],["key=val"]]
 Error NotFound thrown for not-found-example.com
+Error NotFound thrown for not-found-example.com
```

Comment:
I would expect to see something different here. This test should be validating unsupported record type behavior.

Reply:
I'm not sure what the message should look like, could you provide me some?

Reply:
The text here was created on lines 69-73 of the test added in this PR. Based on what this PR is supposed to be changing, I would expect the message to indicate that the record type is not supported. The message should ideally come from the runtime itself and not a string in the test file.